src/towncrier/create.py

```diff
@@ -134,19 +182,16 @@ def __main(
     if filename_parts[-1] in config.types and filename_ext:
         filename += filename_ext
 
-    if config.directory:
-        fragments_directory = os.path.abspath(
-            os.path.join(base_directory, config.directory)
-        )
-    else:
-        fragments_directory = os.path.abspath(
-            os.path.join(
-                base_directory,
-                config.package_dir,
-                config.package,
-                "newsfragments",
+    if not section:
+        if default_section is None:
+            raise click.UsageError(
+                "Multiple sections defined in configuration file, all with paths."
```

Comment:
This looks like a big inconvenient 

Can we just fallback to the first section ?

It will act as a kind of default.

Again, I don't use the `create` command.

So maybe this suggestion makes no sense :)

--------

Also, maybe we can have messages without `please` - https://developers.google.com/style/tone#politeness-and-use-of-please

It's just extra text to read ... keep it short and to the point :)

Reply:
We're using the first section with an empty path as the default, or if all have paths, then the first defined one.